### ISSUE TITLE
[Lead Space Block]: QA: Icon information is not conveyed to the screen reader user.

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9859]
### Description

Page Element:
icon near view all product link

Issue Description:
There is an icon near these link which indicates that these links downloads the file 
The same information is not conveyed  for the screen reader user.
Note: this is applicable for all the variants of lead space."

Recommend Fix:
The information conveyed visually should be available for the screen reader user as well. Anyone of the following methods can be used to make it screen reader accessible.
1. provide the information in the link text. 
2. title attribute. or 
3. as an off screen text
a generic example: 
1. feed back form opens in new window
2. products opens external link

User Impact:
When information is conveyed only through icons, images or graphics and has no appropriate text alternative, screen reader users are unable to understand it.

Test Tool:
JAWS FF
Voiceover safari iPhone

### Component(s) impacted

Lead space block- Video

### Browser

Safari, Firefox

### Carbon for IBM.com version

v1.40.0-rc.0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component staging https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
